tracker/test/plausible-npm.spec.js

```diff
@@ -60,4 +70,19 @@ test.describe('NPM package', () => {
       await callInit(page, DEFAULT_CONFIG, 'window')
     }).rejects.toThrow("plausible.init() can only be called once")
   })
+
+  test('binds to window by default', async ({ page }) => {
+    await openPage(page, {}, { skipPlausibleInit: true })
+    await callInit(page, DEFAULT_CONFIG, 'window')
+    await page.waitForFunction('window.plausible !== undefined')
+  })
+
+  test('does not bind to window if bindToWindow is false', async ({ page }) => {
+    await openPage(page, {}, { skipPlausibleInit: true })
+    await callInit(page, { ...DEFAULT_CONFIG, bindToWindow: false }, 'window')
+    await doesNotResolveWithin(
+      page.waitForFunction('window.plausible !== undefined'),
+      1000
+    )
+  })
```

Comment:
nitpick, non-blocking: I think we can assert the same thing directly with testing APIs directly without extra utils
```suggestion
    await openPage(page, {}, { skipPlausibleInit: true })
    await callInit(page, { ...DEFAULT_CONFIG, bindToWindow: false }, 'window')
    await expect(
      page.waitForFunction(() => window.plausible !== undefined, undefined, { timeout: 1000 }),
    ).rejects.toThrow('page.waitForFunction: Timeout 1000ms exceeded.')
  })
```